### PR TITLE
Specify runAsUser in securityContext for hydra-maester crd job

### PR DESCRIPTION
**Pull request by JackCheng01**
modified:   resources/ory/charts/hydra/charts/hydra-maester/templates/crd-job.yaml

**Description**

Changes proposed in this pull request:

- Specify runAsUser as non-root user in securityContext for hydra-maester crd job

**Related issue(s)**

### CODE REVIEW

File: resources/ory/charts/hydra/charts/hydra-maester/templates/crd-job.yaml [hunk 1]
```diff
@@ -15,6 +15,8 @@ spec:
         sidecar.istio.io/inject: "false"
     spec:
       serviceAccountName: {{ .Release.Name }}-crd-init
+      securityContext:
+        runAsUser: 2000
```
Comment (piotrmsc) [inline resources/ory/charts/hydra/charts/hydra-maester/templates/crd-job.yaml]: Please make this configurable

**Comment by piotrmsc** [comment]
Hi @JackCheng01 Thx for the contribution  🎉  I have left a comment

File: resources/ory/charts/hydra/charts/hydra-maester/templates/crd-job.yaml (hunk 1 above)
Comment (JackCheng01) [reply resources/ory/charts/hydra/charts/hydra-maester/templates/crd-job.yaml]: Yes, we can use variables instead. 
Will check and update this PR.

**Comment by piotrmsc** [comment]
@JackCheng01 any updates?

**Comment by pbochynski** [comment]
Closing as there is no further input.

**Pull request closed without merging**